### PR TITLE
Update to SQLitePCLRaw 2.1.2-pre20220913165057

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -13,6 +13,8 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <!-- TODO: Mirror packages on dotnet-public -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (bricelam) [inline NuGet.config:17]: @bricelam Don't forget.

**Comment by ajcvickers** [comment]
@bricelam Should we be doing this for 7.0? We'll need to make sure we have a stable release in time for the GA build.

**Comment by bricelam** [comment]
This SQLitePCL release adds support for Wasm on .NET 7. I think it's a risk either way, so I'll let you make the call.

Worst case scenario if we don't take this is that users will have to manually update the SQLitePCLRaw.bundle_e_sqlite3 package in order to get Wasm to work.

@ericsink Do you have an estimate for a non-prerelease?

**Comment by ericsink** [comment]
I can do a non-pre release anytime you want, including today.  I was stalling to see if clarity emerged on a couple of other issues, but there's nothing there yet, so the updated Wasm binaries would be the only effective change since 2.1.1.

**Comment by bricelam** [comment]
Note, this will resolve the issue originally reported as https://github.com/dotnet/runtime/issues/74988.

**Comment by ajcvickers** [comment]
Let's do it!

**Review by ajcvickers — APPROVED** [review]

**Comment by bricelam** [comment]
@ericsink We're branching for RC2 soon, so a prerelease is still OK. At the latest we'll probably need a final version by like Oct 7, so no rush. As always though, the sooner the better, so we have time to react to any last-minute surprises.

**Pull request merged**